### ISSUE TITLE
Add oneWire flash memory support

### ISSUE THREAD

**Issue by c0t0d0s0** [spectre338/IoT_Porter#3]
A small EEPROM like the DS28E05 would allow the mapping of serial number to sensor number to happen outside of the code. It could be installed as the last device in the sensor string and easily be reprogrammed if a sensor needs to be replaced without dePotting the ESP.

**Comment by spectre338** [comment]
I agree the bus does not know what order the sensors are. They index on a FIFO model; the first test was using index and the string assembled as 2>4>1>5>2 in physical position. This is problematic as probe1 needs to be roof, probe2 top third, probe3 halfway, probe4 lower third, and probe5 6 inches underground

**Comment by spectre338** [comment]
Also will potting be necessary as the will installed in [https://www.amazon.com/Awclub-Waterproof-Dustproof-Universal-83mmx58mmx33mm/dp/B07CG27FWX/](url) ?

The USB Type C will be available via [https://www.amazon.com/ANMBEST-Mounting-Waterproof-Connector-PA6BG33/dp/B09QLR3ZG6](url)

As for the string of sensors and solar panel will either be a gland or [https://www.amazon.com/Waterproof-Aviation-Connector-connector-Circular/dp/B0878F7WLT](url)
